Dockerfile

```diff
@@ -1,13 +1,19 @@
 # example showing sudo config
-# docker run -it -p 9000:8000 jupyter/jupyterhub-sudo
+# docker run -it -p 9000:8000 jupyterhub/jupyterhub-sudo
 
-FROM jupyter/jupyterhub
+FROM jupyterhub/jupyterhub:0.8.0
 
 MAINTAINER Jupyter Project <jupyter@googlegroups.com>
 
+RUN apt-get -y update \
+    && apt-get -y install sudo \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 # fix permissions on sudo executable (how did this get messed up?)
 RUN chmod 4755 /usr/bin/sudo
 
+RUN python3 -m pip install notebook
+
 # add the rhea user, who will run the server
 # she needs to be in the shadow group in order to access the PAM service
 RUN useradd -m -G shadow -p $(openssl passwd -1 rhea) rhea
```

Comment:
Outside of the scope of this PR but I wonder if moving line 19 and replacing line 13 with it would correct the permissions without a chmod?

Reply:
I'm not sure! I didn't check the permissions on sudo. Since this is a new base image, there's a very good chance that sudo permissions don't need fixing anymore.